videos/WoT-Explainer-Video/WOTMotion.fr.vtt

```diff
@@ -0,0 +1,251 @@
+WEBVTT
+
+1
+00:00:00.500 --> 00:00:03.000
+Un producteur fruitier cherche des moyens
+pour augmenter sa productivité 
+
+2
+00:00:03.500 --> 00:00:06.000
+tout en conservant une excellente
+qualité de produits. 
+
+3
+00:00:06.500 --> 00:00:08.000
+Pour surveiller et contrôler
+les conditions de croissance,
+
+4
+00:00:08.500 --> 00:00:13.000
+l'agriculteur prévoit d'installer des capteurs d'humidité et de température,
+et un système d'arrosage.
+
+5
+00:00:13.500 --> 00:00:16.000
+Les données de prévisions météorologiques
+à partir d'un service Web en ligne 
+
+6
+00:00:16.500 --> 00:00:20.000
+seraient également utile pour
+pour prévoir les besoins en arrosage.
+
+7
+00:00:20.500 --> 00:00:25.000
+L'agriculteur décide d'engager
+un consultant informatique. 
+
+8
+00:00:25.500 --> 00:00:29.000
+Il voudrait demander au développeur de son site Web
+de réaliser une solution IdO.
+
+9
+00:00:29.500 --> 00:00:33.000
+Les développeurs à l'agence Web
+aiment ce nouveau défi 
+
+10
+00:00:29.500 --> 00:00:36.000
+et sont prêts à élargir
+leurs connaissances en IdO.
+
+11
+00:00:36.500 --> 00:00:41.000
+Ils sont experts en technologies frontend et 
+backend utilisant Node JS.
+
+12
+00:00:41.500 --> 00:00:45.000
+Cependant, ils s'inquiètent
+des problèmes d'interopérabilité 
+
+13
+00:00:45.500 --> 00:00:49.000
+dont ils ont fait l'expérience
+lors d'un précédent projet IdO.
+
+14
+00:00:49.500 --> 00:00:53.000
+Après une recherche en ligne
+pour une solution,
+
+15
+00:00:53.500 --> 00:00:58.000
+les développeurs trouvent des informations
+sur le Web des objets, WoT 
+
+16
+00:00:58.500 --> 00:01:03.000
+WoT est une norme W3C qui permet aux développeurs Web
+de devenir des développeurs IdO 
+
+17
+00:01:03.500 --> 00:01:08.000
+en utilisant les outils
+et les systèmes qu'ils aiment. 
+
+18
+00:01:08.500 --> 00:01:12.000
+WoT peut être appliqué
+à de nombreux domaines IdO,
+
+19
+00:01:12.500 --> 00:01:15.000
+tels que la fabrication
+ou l'immotique, 
```

Comment:
"immotique" ? Je ne connaissais pas ce terme (mais ne conteste pas qu'il existe !). Une traduction plus fidèle de "Building automation" serait "gestion technique de bâtiment".

Reply:
> commented FR translation, as a French-speaker working with WoT technologies.

I think each of @vcharpenay's suggestions should be turned into changes (I don't know how to do that myself, otherwise I would.) His credentials "French speaker + working with WoT technologies" clearly trump mine (French speaker).

Reply:
Not sure if i got the comment @koalie  ...
To update your proposal you can simple go to https://github.com/w3c/wot-marketing/pull/152/files click on the 3 dots and go with "edit file" and do the changes accordingly...

Reply:
Perfect, thanks @danielpeintner that is exactly the guidance I was looking for. I have done so throughout the file now.